docs/buildspecs/e4s_testsuite.rst

```diff
@@ -0,0 +1,49 @@
+.. Note:: Please see :ref:`tutorial_setup` before you proceed with this section
+
+.. _buildtest_e4s_testsuite:
+
+Buildtest E4S Testsuite Integration
+===================================
+
+The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
+software products provided by `E4S <https://e4s.io>`_ deployments. 
+E4S is a spack based software stack but because not all spack packages provide internal testing functionality it 
+is useful to have an alternative framework for testing deployed software.
```

Comment:
i think this sentence  can be removed, we don't need to explain to user why we need an alternative framework for testing spack packages.

Reply:
Actually, this question comes up a lot in discussions about Spack Tests/E4S Testsuite.